app/screens/channel/channel.tsx

```diff
@@ -149,6 +150,8 @@ const Channel = ({
                             nativeID={channelId}
                         />
                     </View>
+                    {/*This count is hardcoded but will be removed during integration work*/}
```

Comment:
Can you add a TODO: on it?

Reply:
Done.

Reply:
i've added the TODO and the corresponding eslint fix anyways. LMK if you want me to continue adding those.